Client/Frontend/Browser/TabManager.swift

```diff
@@ -857,7 +861,8 @@ extension TabManager {
         isRestoring = false
 
         // Always make sure there is a single normal tab.
-        if normalTabs.isEmpty {
+        let tabs = self.tabs(withType: .regular)
```

Comment:
Eventually this will need to be refactored, `self.tabs` should just return the valid tabs, regardless of the "mode". But this may be out of scope for this PR.

Reply:
I think it is best that this work is part of mixed tabs, I will raise a ticket